apps/aeutils/src/aeu_env.erl

```diff
@@ -305,23 +310,34 @@ read_config(Mode) when Mode =:= silent; Mode =:= report ->
 
 apply_os_env() ->
     ok = application:ensure_started(gproc),
-    try
-    %% Make sure gproc is started, since we use it in update_config/1
-    application:ensure_started(gproc),
     Pfx = "AE",  %% TODO: make configurable
```

Comment:
shouldn't this be `DEV` or so then ?

Reply:
This function is called during the normal boot. The support function, called from `aeu_plugins` is flexible in regard to prefix and schema. The thing that is not yet configurable is the os env prefix for overriding the parameters in the main schema.